.gitpod/tasks/2-app-dev/command.sh

```diff
@@ -4,4 +4,12 @@ set -e
 cd webstone-dev-app
 while [ ! -f ../webstone/packages/cli/dist/bin.js ]; do sleep 1; done
 pnpm add -D ../webstone/packages/cli
+
+# Install all plugin-*/cli packages
+for FILE in $(find ../webstone/packages/plugin-*/cli/package.json)
+do
+  DIR=`dirname $FILE`
+  pnpm add -D $DIR
+done
```

Comment:
This installs all `plugin-*/cli` packages automatically in the `webstone-dev-app`. So when you run `pnpm ws` in the dev app, you have access to all plugins.

This gives us hot-module reloading for the CLI plugins! Make a code change to a plugin's CLI and test it in the dev app instantly.